docs/spec_draft.md

```diff
@@ -1576,6 +1578,48 @@ More formally, for each element `x`: `imag(x) = is_complex(x) ? x.imag : 0.0`.
 
 [Back to Ops](#index-of-ops)
 
+
+## stablehlo.infeed
+
+### Semantics
+
+Reads `results` from the infeed queue of the device. Multiple
+`stablehlo.infeed` operations are allowed in a computation, but there must be a
+total order among them.
+
+The infeed configuration `infeed_config` includes target-dependent metadata
```

Comment:
`infeed_config` is an arbitrary sting which may include target-dependent metadata.

Reply:
Just keeping this unresolved for the next reviewer. I am not sure saying `infeed_config` arbitrary string would be right, but I would still go with this or maybe just remove the word arbitrary.

`infeed_config` is a sting which may contain target-dependent metadata.